config/config.yml

```diff
@@ -8,6 +8,7 @@ metadata:
   siteImage:
   favicon: "/assets/favicon.png"
   themeColor: "#ff0000"
+  ogImage: "/assets/dotnetthailand-ogimage-dark.png"
```

Comment:
@surrealist  Just curiosity, how the current system works without adding ogImage.
I wonder if we don't need to add ogImage here.

Reply:
I pushed new commit with detailed explanation.